app/javascript/gobierto_TEMP/webapp/Maker.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <div class="dashboards-maker">
+    <HeaderForm
+      :is-dirty="dirty"
+      :config="configuration"
+      @save="handleSave"
+      @edit="handleEdit"
+      @delete="handleDelete"
+      @view="handleView"
+    >
+      <template #title>
+        <TextEditable
+          tag="h1"
+          :icon="true"
+          @input="handleInputTitle"
+        >
+          {{ title }}
+        </TextEditable>
+      </template>
+
+      <template #checkbox>
+        <label
+          class="dashboards-maker--checkbox"
+          for="dashboard-status"
+        >
+          <input
+            id="dashboard-status"
+            type="checkbox"
+            @input="handleInputStatus"
+          >
+          {{ publicLabel }}
+        </label>
+      </template>
+    </HeaderForm>
+
+    <Main>
+      <template #aside>
+        <Aside>
+          <SmallCard
+            v-for="({ name }, key) in cards"
+            :key="key"
+            :name="name"
+            @drag.native="drag(key, $event)"
+            @dragend.native="dragend"
+          />
+        </Aside>
+      </template>
+
+      <Viewer
+        v-if="configuration"
+        ref="viewer"
+        :is-draggable="true"
+        :is-resizable="true"
+        :item="item"
+        :config="configuration"
+        @layout-updated="handleViewerUpdated"
+        @layout-ready="handleViewerReady"
+      />
+    </Main>
+  </div>
+</template>
+
+<script>
+// add the styles here, because this element can be inserted both as a component or standalone
+import "../../../assets/stylesheets/module-TEMP-maker.scss";
+import Viewer from "./Viewer";
+import HeaderForm from "./components/HeaderForm";
+import Aside from "./layouts/Aside";
+import Main from "./layouts/Main";
+import SmallCard from "./components/SmallCard";
+import { Widgets } from "./lib/widgets";
+import { DashboardFactoryMixin } from "./lib/factories";
+import { TextEditable } from "lib/vue-components";
+
+export default {
+  name: "Maker",
+  components: {
+    Viewer,
+    HeaderForm,
+    Main,
+    Aside,
+    SmallCard,
+    TextEditable,
+  },
+  mixins: [DashboardFactoryMixin],
+  data() {
+    return {
+      dirty: false,
+      cards: Widgets,
+      item: null,
+      configuration: null,
+      viewerLoaded: false,
+      publicLabel: "Público",
```

Comment:
locale

Reply:
I gonna create an issue (aside from this one) to move all the locale texts. That was, because initially I didn't want to create a config/locale/gobierto_temp, until the renaming was made.

Reply:
I resolve other related conversations, ok?

Reply:
Perfect!